docs/faq.rst

```diff
@@ -124,10 +124,34 @@ Which ML libraries does Fairlearn support?
     `predict()` methods. Also, any classification or regression
     algorithm can be evaluated using our metrics.
 
+Does Fairlearn support multi-class classification?
+    On the assessment side, Fairlearn's :class:`MetricFrame` can be used with
+    any metrics for supervised learning, including those for multi-class classification.
+    For example, it is possible to pass
+    :py:func:`sklearn.metrics.accuracy_score` or :py:func:`sklearn.metrics.confusion_matrix`
+    as the metric functions, and supply multi-class data for :code:`y_true` and :code:`y_pred`.
+    There are
+    `ongoing discussions within the community <https://github.com/fairlearn/fairlearn/issues/752>`_
+    to add more extensive support to Fairlearn's assessment capabilities.
+    If you have thoughts feel free to add them to the discussion.
+    
+    On the mitigation side, our algorithms :class:`ExponentiatedGradient` and :class:`GridSearch`
```

Comment:
We have never tested this, but this should be allowed. I realize that our user guide doesn't say this... we can fix that another time.